### PR TITLE
Basic matrix view for portfolio users

### PR DESCRIPTION
**Pull request by pkmitre**

### CODE REVIEW

**Comment by thecristen** [comment]
Very cool!
One thing I noticed is that, when no expectations are set _and_ the patient is in none of the populations, that patient shows as failing in the matrix view. It seems a little inconsistent having those patients show as 'problem' in the matrix view but 'pass' within that measure's view.

**Comment by pkmitre** [comment]
The matrix should show a blank in the case you describe, and that's what I see on my system. When you get a chance, can you show me what it looks like for you?

**Comment by thecristen** [comment]
Actually you’re right.. Ok, must’ve been a mistake on my eyes then.

File: app/assets/javascripts/views/matrix_view.js.coffee [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+class Thorax.Views.MatrixCell extends Thorax.Views.BonnieView
+
+  tagName: 'td'
+
+  template: -> null # Empty template, everything is in the class name
+
+  events:
+    rendered: ->
+      if @model.differenceFromExpected().has('match')
+        popoverContent = JST['population_calculation_results'](patient: @model.patient.toJSON(), comparisons: @model.differenceFromExpected().get('comparisons'))
```
Comment (jtferns) [inline app/assets/javascripts/views/matrix_view.js.coffee:27]: The styles for the population calculation results are embedded within <code>measure_calculation.less</code>.

If we move the following out (to somewhere like <code>mixins.less</code>; or embed the same styles within <code>matrix.less</code>) we can get styles on the popovers, too!

```
.pass {
  .text-success;
}
.fail {
  .text-danger;
}
.default {
  color: @brand-primary;
}
```

screen shot 2014-08-15 at 10 14 54 am

File: app/assets/javascripts/views/matrix_view.js.coffee [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        @$el.popover trigger: 'hover', placement: 'bottom', container: 'body', title: @model.population.displayName(), html: true, content: popoverContent
```
Comment (jtferns) [inline app/assets/javascripts/views/matrix_view.js.coffee]: Optionally, having the patient name in the popover title might be helpful when looking at results on the far right and the patients column is not within view.

screen shot 2014-08-15 at 10 24 26 am

**Comment by pkmitre** [comment]
Good suggestions, I'll get them both in.

**Pull request merged**